docs/rules/prefer-hooks-on-top.md

```diff
@@ -1,7 +1,7 @@
 # Suggest having hooks before any test cases (`prefer-hooks-on-top`)
 
-While hooks can be setup anywhere in a test file, they are always called in a
```

Comment:
weird flow without this one to me. but I'm not a native speaker, so feel free to ignore 😀

Reply:
I agree with you on this

Reply:
Whoops, missed that one, re-added in https://github.com/jest-community/eslint-plugin-jest/pull/1086/commits/0cc62f8a21512c43a6fdb9ab38464ede65b03a1c